src/service/BookingService.java

```diff
@@ -5,26 +5,37 @@
 import java.time.LocalDateTime;
 import config.Setting;
 import database.dao.BookingDataDAO;
+import database.dao.GuestDataDAO;
 import database.dto.BookingDataDTO;
+import database.dto.GuestDataDTO;
+import database.dto.NationalityDTO;
 import database.dto.PaymentMethodDTO;
 
 public class BookingService {
 	
 	private final BookingDataDAO bookingDataDAO;
+	private final GuestDataDAO guestDataDAO;
 	private final Setting setting;
 	
-	
 	public BookingService() {
 		this.bookingDataDAO=new BookingDataDAO();
+		this.guestDataDAO = new GuestDataDAO();
 		this.setting = new Setting();
 	}
-
-	public BookingService(BookingDataDAO bookingDataDAO) {
+	
+	public BookingService(BookingDataDAO bookingDataDAO,GuestDataDAO guestDataDAO) {
 		this.bookingDataDAO=bookingDataDAO;
+		this.guestDataDAO = guestDataDAO;
 		this.setting = new Setting();
 	}
 	
-	public Integer savedBooking(LocalDateTime entryDate, LocalDateTime departureDate, PaymentMethodDTO methodPayment)  {
+	public BookingService(GuestDataDAO guestDataDAO) {
```

Comment:
I don't see big purpose for this constructor, and you aren't using it, please remove, unless there is a purpose for it.